content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md

```diff
@@ -96,42 +99,42 @@ The upgrade workflow at high level is the following:
     [preflight] Running pre-flight checks.
     [upgrade] Making sure the cluster is healthy:
     [upgrade] Fetching available versions to upgrade to
-    [upgrade/versions] Cluster version: v1.14.2
-    [upgrade/versions] kubeadm version: v1.15.0
+    [upgrade/versions] Cluster version: v1.15.2
+    [upgrade/versions] kubeadm version: v1.16.0
 
     Components that must be upgraded manually after you have upgraded the control plane with 'kubeadm upgrade apply':
     COMPONENT   CURRENT       AVAILABLE
-    Kubelet     1 x v1.14.2   v1.15.0
+    Kubelet     1 x v1.15.2   v1.16.0
 
-    Upgrade to the latest version in the v1.15 series:
+    Upgrade to the latest version in the v1.16 series:
 
     COMPONENT            CURRENT   AVAILABLE
-    API Server           v1.14.2   v1.15.0
-    Controller Manager   v1.14.2   v1.15.0
-    Scheduler            v1.14.2   v1.15.0
-    Kube Proxy           v1.14.2   v1.15.0
-    CoreDNS              1.3.1     1.3.1
-    Etcd                 3.3.10    3.3.10
+    API Server           v1.15.2   v1.16.0
+    Controller Manager   v1.15.2   v1.16.0
+    Scheduler            v1.15.2   v1.16.0
+    Kube Proxy           v1.15.2   v1.16.0
+    CoreDNS              1.3.1     1.6.2
+    Etcd                 3.3.10    3.3.15
 
     You can now apply the upgrade by executing the following command:
 
-        kubeadm upgrade apply v1.15.0
+        kubeadm upgrade apply v1.16.0
 
     _____________________________________________________________________
     ```
 
     This command checks that your cluster can be upgraded, and fetches the versions you can upgrade to.
 
     {{< note >}}
-    With the release of Kubernetes v1.15, `kubeadm upgrade` also automatically renews
+    With the release of Kubernetes v1.16, `kubeadm upgrade` also automatically renews
```

Comment:
I think we can safely remove "with the release of Kubernetes v1.16" now. This will simplify future updates of this doc..